src/odemis/driver/simcam.py

```diff
@@ -42,14 +42,14 @@ class Camera(model.DigitalCamera):
     given at initialisation.
     '''
 
-    def __init__(self, name, role, image, dependencies=None, daemon=None, blur_factor=1e4, **kwargs):
-        '''
+    def __init__(self, name, role, image, dependencies=None, daemon=None, blur_factor=1e4, max_res=None, **kwargs):
+        """
         dependencies (dict string->Component): If "focus" is passed, and it's an
             actuator with a z axis, the image will be blurred based on the
             position, to simulate a focus axis.
-        image (str or None): path to a file to use as fake image (relative to
-         the directory of this class)
-        '''
+        image (str or None): path to a file to use as fake image (relative to the directory of this class)
+        max_res (tuple of (float, float) or None): maximum resolution to clip simulated image, if None whole image shape will be used
```

Comment:
It'd better be a (int, int), instead of (float, float). Also, please indicate that the simulated image will be a part of the original image based on the MD_POS metadata.